SIL.Core/IO/FileUtils.cs

```diff
@@ -334,5 +335,39 @@ public static string NormalizePath(string path)
 		{
 			return path.Replace('\\', '/');
 		}
+
+		/// <summary>
+		/// Strips file URI prefix from the beginning of a file URI string, and keeps
+		/// a beginning slash if in Linux.
+		/// eg "file:///C:/Windows" becomes "C:/Windows" in Windows, and
+		/// "file:///usr/bin" becomes "/usr/bin" in Linux.
+		/// Returns the input unchanged if it does not begin with "file:".
+		///
+		/// Does not convert the result into a valid path or a path using current platform
+		/// path separators.
+		///
+		/// See uri.LocalPath, http://en.wikipedia.org/wiki/File_URI , and
+		/// http://blogs.msdn.com/b/ie/archive/2006/12/06/file-uris-in-windows.aspx .
+		/// </summary>
+		public static string StripFilePrefix(string fileString)
```

Comment:
I would have suggested StripUriFilePrefix for clarity, but as this has been merged. (shrug)